low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -54,7 +55,7 @@ def usage(prog: str):
             [ post_install --config [<global_config_url>] ]
             [ init --config [<global_config_url>] ]
             [ config --config [<global_config_url>] ]
-            [ test [sanity|alerts] ]
+            [ test [sanity|alerts|self_primary|self_secondary] ]
```

Comment:
mention which plan would be executed by default, if no argument given.

Provisioner also mentioned to use --config <global_config_url> for test too, which we dont need as of today. So keep a provision to accept that too, and not use it, will see if future.
@sandeepsgt , your PR taking care of following i believe
During sspl_tests, controller IPs & some config (in global set) are updated in backend, are we making copy of global config and then updating/using that copy for test. Asking because it wont be good to update global config directly, today its fine from sample global config, but not from provisioner supplied global config.

Reply:
@cdeshmukh currently we are keeping the validator if no plan is passed, it will raise a SetupError:-
if not self.args:
    raise SetupError(1,
                             "Validation failure. %s",
                             "Test requires the plan type(sanity|alerts \
                             |self_primary|self_secondary).")

Reply:
`are we making copy of global config and then updating/using that copy for test`  Not exactly, but kind of
Current flow
- backup conf files 
- Update original files 
- Restore backup

Reply:
Added "self_primary" plan as default. Please check.